tide_authenticated_content.install

```diff
@@ -25,19 +25,6 @@ function tide_authenticated_content_install() {
       'permissions_by_term',
       'rest',
     ]);
-
-    // Create taxonomy.
```

Comment:
Are these actions now not required, or covered by other code in other modules for new installs?

Reply:
it should get all these from the [install directory ](https://github.com/dpc-sdp/tide_authenticated_content/tree/feature/SRM-877-d10-update/config/install) for fresh install and new projects.